From: Vincent Bernat <vincent@bernat.im>
Date: Sat, 22 Aug 2015 19:28:13 +0200
Subject: Avoid fetching jQuery from Google, use the embedded one

This page is also just an example. The user is expected to provide their
own page.

Forwarded: not-needed
---
 plugins/http_authentication/logout.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/http_authentication/logout.html b/plugins/http_authentication/logout.html
index 30a28ef..bc08086 100644
--- a/plugins/http_authentication/logout.html
+++ b/plugins/http_authentication/logout.html
@@ -3,7 +3,7 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <title>Logout</title>
-<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6/jquery.min.js"></script>
+<script type="text/javascript" src="../../program/js/jquery.min.js"></script>
 <script type="text/javascript">
 
 // as seen on http://stackoverflow.com/questions/31326/is-there-a-browser-equivalent-to-ies-clearauthenticationcache
